src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,99 @@
+package duke;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The TaskList class contains a list of tasks and keeps track of it while Duke is running.
+ */
+public class TaskList {
+
+    private List<Task> tasks;
+    private static String BORDER = "-----------------------------------------------------------";
```

Comment:
Full caps naming is for immutable constants, you should perhaps include "final" to make these values immutable.